tools/check/ut.go

```diff
@@ -359,12 +367,12 @@ func (n *numa) runTestCase(pkg string, fn string, old bool) error {
 		cmd = n.testCommand(exe, fn, old)
 	}
 	cmd.Dir = path.Join(workDir, pkg)
-	_, err := cmd.CombinedOutput()
-	if err != nil {
-		// fmt.Println("run test case error", pkg, fn, string(output))
-		return err
+	cmd.Stdout = &res.output
```

Comment:
Is this related to your PR description? Please add a comment why you want to handle `Stdout` & `Stderr` here.

Reply:
> Modification on tools/check/ut.go is unrelated to this fix, it just make the output easier for debugging.

Reply:
In the past, the stdout/stderr of the test result is ignored, so it's not convenient to debug
The tool `ut` is still evolving, before one day it can be made in to default CI pipeline.